neptune/new/logging/handler.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2021, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import logging
+
+from neptune.new import Run
+from neptune.new.attributes.constants import MONITORING_DEFAULT_LOG_HANDLER_ATTRIBUTE_PATH
+from neptune.new.internal.utils import verify_type
+from neptune.new.logging import Logger
+
+
+class NeptuneLogHandler(logging.Handler):
```

Comment:
The naming is incorrect with regard to the specification.
1. In general the naming from the specification should be a source of truth, and let's keep all the discussions about naming, public API etc. there. They are a form of "design spec" and should be treated similar :)
2. On the other way this prompted me to review the naming there it should be the following:
`from neptune.new.integrations.python_logger import NeptuneHandler`
a) what the thing is about we capture in module name (in this case *integration* with *python logger*)
b) we follow the naming of what we integrate with prefixed with Neptune. In this case all logger naming points to Handler

Reply:
(I've update the spec accordingly)

Reply:
sorry, I'll voice my opinions in design spec next time